src/miniscript/mod.rs

```diff
@@ -168,7 +171,38 @@ impl<Ctx: ScriptContext> Miniscript<bitcoin::PublicKey, Ctx> {
     /// This function will fail parsing for scripts that do not clear
     /// the [Miniscript::sanity_check] checks. Use [Miniscript::parse_insane] to
     /// parse such scripts.
-    pub fn parse(script: &script::Script) -> Result<Miniscript<bitcoin::PublicKey, Ctx>, Error> {
+    /// ## Decode/Parse a miniscript from script hex
```

Comment:
Why from "script hex" when we provide the method with a `Script` object, not a hex string?

Reply:
Oops, just realized that the example below shows decoding from hex. That is why it was labelled as such